frontend/src/App.jsx

```diff
@@ -52,8 +52,7 @@ function App() {
             element={user ? <MyTeamView /> : <LoginSignup loginForm={true} />}
           ></Route>
           <Route
-            exact
-            path="/player"
+            path="/player/:id"
```

Comment:
https://github.com/GloriaXM/numbers-game/assets/112780860/4a199e84-1b24-4ae9-83e1-724bcd03a014

Can now refresh the page without losing what player is currently being viewed.